src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/CrimeBlobDispatchingTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.blobrouter;
+
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.toByteArray;
+import static com.jayway.awaitility.Awaitility.await;
+import static java.util.Arrays.asList;
+import static uk.gov.hmcts.reform.blobrouter.envelope.ZipFileHelper.createZipArchive;
+import static uk.gov.hmcts.reform.blobrouter.storage.StorageHelper.blobExists;
+import static uk.gov.hmcts.reform.blobrouter.storage.StorageHelper.uploadFile;
+
+public class CrimeBlobDispatchingTest extends FunctionalTestBase {
+
+    private BlobServiceClient crimeStorageClient;
+
+    @BeforeEach
```

Comment:
`FunctionalTestBase` also has `@BeforeEach` - it works out?

Reply:
Well spotted, fixed